### PR TITLE
add extended all.sh to allow [extended:all] to run all buckets

### PR DESCRIPTION
**Pull request by deads2k**
@mfojtik simple, naive, early exit-ing, but helpful.

### CODE REVIEW

**Comment by deads2k** [comment]
[testextendedonly][extended:all]

**Comment by mfojtik** [comment]
like this. we should run this daily in jenkins.

**Comment by deads2k** [comment]
[testextendedonly][extended:all]

**Comment by deads2k** [comment]
re[test]

**Comment by deads2k** [comment]
the test failed, but it did what it was supposed to do.

**Comment by deads2k** [comment]
Note that this will require an update to the openshift-vagrant plugin to call `all.sh` instead of calling `*.sh` scripts.

**Comment by danmcp** [comment]
@deads2k Bump

**Comment by deads2k** [comment]
re[test]

@mfojtik I still want this.  You?

**Comment by mfojtik** [comment]
yes would be nice to have and run as release verification

**Comment by deads2k** [comment]
re[test]

**Comment by deads2k** [comment]
@stevekuznetsov ptal

**Comment by deads2k** [comment]
@stevekuznetsov the run was a failure because extended/core failed, but it looks like the intent worked.  ptal.

File: test/extended/all.sh [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+test_scripts=`find test/extended -maxdepth 1 -name "*.sh" -not  \( -name "all.sh" \)`
+
+OVERALL_RETURN=0
+for test_script in $test_scripts; do
+	STARTTIME=$(date +%s)
+	echo "${STARTTIME} starting ${test_script}";
+
+	set +e
+	(${test_script})
```
Comment (stevekuznetsov) [inline test/extended/all.sh]: leave an explanatory comment about the subshell here

**Comment by stevekuznetsov** [comment]
tiny nit, LGTM

**Comment by deads2k** [comment]
[merge]

I will update the jenkins extended job after this merges.

**Pull request merged**